application/src/main/data/upgrade/3.4.4/schema_update.sql

```diff
@@ -423,6 +423,17 @@ $$;
 
 -- ALARM FUNCTIONS END
 
+
+-- DEVICE PROFILE CERTIFICATE START
+
+ALTER TABLE device_profile
+    ADD COLUMN IF NOT EXISTS certificate_hash varchar,
```

Comment:
provisionDeviceKey already exists and can be used to achieve the same result, without adding extra field and constraint to the table.

Reply:
@imbeacon this is a bit different. It is hard to explain why we store the certificate hash of the device profile in the provisionDeviceKey field. Bad naming for this.